requirements.txt

```diff
@@ -1,5 +1,4 @@
 ckantoolkit>=0.0.3
 goodtables==1.5.1
 six>=1.13.0
-pyrsistent<0.17.0
```

Comment:
why not required?

Reply:
There's nothing that imports and uses it.